README.md

```diff
@@ -382,18 +400,20 @@ When the installation finishes, Docker starts automatically. The whale   in the
 ### Instructions to run docker
 1. Create an .env file with variables in the following format, each on a new line:
    ```bash
-      POSTGRES_USER=postgres
-      POSTGRES_PASSWORD=postgres
-      POSTGRES_DB=db
+   #!/usr/bin/env bash
```

Comment:
```suggestion
```
This is not a bash script, it is actually a .env file. So while we have it syntax formatted as bash it should not be run using bash. Remove this shebang